images/sprite.svg

```diff
@@ -0,0 +1,72 @@
+<svg aria-hidden="true" style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
please add a comment to explain how we can generate this file in the future.

Reply:
i was hoping to write a small wiki for it, where do you think I should write? a blog post would be overkill.

Reply:
Please use this one: 
https://github.com/deepnetworkgmbh/deepnetworkgmbh.github.io/wiki/Generating-sprite.png-for-icons